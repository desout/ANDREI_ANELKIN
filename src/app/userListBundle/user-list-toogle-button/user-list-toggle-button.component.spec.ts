import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { UserListToggleButtonComponent } from './user-list-toggle-button.component';
import {NO_ERRORS_SCHEMA} from '@angular/core';
import {TestModule} from '../../Test.module';

describe('UserListToggleButtonComponent', () => {
  let component: UserListToggleButtonComponent;
  let fixture: ComponentFixture<UserListToggleButtonComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [  ],
      schemas: [NO_ERRORS_SCHEMA],
      imports: [TestModule],
      providers: []
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserListToggleButtonComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
