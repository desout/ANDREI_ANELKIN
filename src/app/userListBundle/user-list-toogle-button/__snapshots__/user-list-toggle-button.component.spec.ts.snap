// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserListToggleButtonComponent should create 1`] = `
<div
  id="root0"
  ng-version="7.0.3"
>
  <div
    class="btn btn-secondary dropdown-toggle"
  />
</div>
`;
